tooling/nargo/src/foreign_calls/mod.rs

```diff
@@ -93,3 +94,14 @@ pub enum ForeignCallError {
     #[error("Failed to replay oracle transcript: {0}")]
     TranscriptError(String),
 }
+
+impl From<TryFromParamsError> for ForeignCallError {
+    fn from(err: TryFromParamsError) -> Self {
+        match err {
```

Comment:
Do both enums have the same variants? If so, could one be an alias of the other?

Reply:
No, ForeingCallError has many more variants